viz/charts/linkerd-viz/templates/tap-injector-rbac.yaml

```diff
@@ -0,0 +1,95 @@
+###
+### Tap Injector RBAC
+###
+---
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: {{.Values.namespace}}-tap-injector
+  labels:
+    {{.Values.extensionAnnotation}}: {{.Values.namespace}}
```

Comment:
The value should be hard-coded to `linkerd-viz` as we might be selecting elsewhere using that value regardless of the namespace used for linkerd-viz. (same comment for the resources below).